src/main/java/com/revature/domain/Batch.java

```diff
@@ -0,0 +1,84 @@
+package com.revature.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="ie_batch")
+public class Batch {
+	
+	@Id
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="batch_sequence")
+	@SequenceGenerator(allocationSize=1, name="batch_sequence", sequenceName="batch_seq")
+	@Column(name="b_id")
+	private int id;
+	
+	@Column(name="b_name")
+	private String name;
+
+	public Batch() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)